### PR TITLE
EN-3836-fix bug with status is synchornized while it shouldn't have been

### PR DESCRIPTION
**Pull request by miiu96**
fix bug with status is synchronized while it shouldn't

### CODE REVIEW

File: cmd/node/main.go [hunk 1]
```diff
@@ -538,6 +538,7 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 	coreComponents.StatusHandler.SetStringValue(core.MetricConsensusState, "")
 	coreComponents.StatusHandler.SetStringValue(core.MetricConsensusRoundState, "")
 	coreComponents.StatusHandler.SetStringValue(core.MetricCrossCheckBlockHeight, "")
+	coreComponents.StatusHandler.SetUInt64Value(core.MetricIsSyncing, uint64(1))
```
Comment (sasurobert) [inline cmd/node/main.go]: what does uint64(1) means ?

Comment (iulianpascalau) [reply cmd/node/main.go]: Should have a constant for that

Comment (miiu96) [reply cmd/node/main.go]: Done.

File: statusHandler/presenter/presenterStatusHandler_test.go [hunk 2]
```diff
@@ -103,14 +103,15 @@ func TestPresenterStatusHandler_Write(t *testing.T) {
 func TestPresenterStatusHandler_GetLogLine(t *testing.T) {
 	t.Parallel()
 
+	waitTimeToWrite := 10 * time.Millisecond
```
Comment (sasurobert) [inline statusHandler/presenter/presenterStatusHandler_test.go]: waitTimeBeforeRead

Comment (miiu96) [reply statusHandler/presenter/presenterStatusHandler_test.go]: Done. Name was changed.

**Pull request merged**